chat-widget/src/common/telemetry/TelemetryHelper.ts

```diff
@@ -308,4 +308,15 @@ export class TelemetryHelper {
         };
         BroadcastService.postMessage(telemetryEvent);
     }
+
+    public static logFacadeChatSDKEvent = (logLevel: LogLevel, payload: TelemetryEventWrapper) => {
+        const telemetryEvent: ITelemetryEvent = {
```

Comment:
Can we just declare it as FacadeChatSDKTelemetryData instead of recasting it?
It seems that `eventName` doesn't exist on that type

Reply:
BroadcastService uses ITelemetryEvent, FacadeChatSDKTelemetryData is used to ensure the contract is enforced